install/kubernetes/templates/istio-mixer.yaml.tmpl

```diff
@@ -554,23 +554,6 @@ spec:
   scope: Namespaced
   version: v1alpha2
 ---
-
-kind: CustomResourceDefinition
-apiVersion: apiextensions.k8s.io/v1beta1
-metadata:
-  name: reportnothings.config.istio.io
```

Comment:
curious: why is reportnothings disappearing?

Reply:
Yeah, expand the unedited parts a few times, you'll see another reportnothings.